### PR TITLE
zh-cn(update): update client-id

### PR DESCRIPTION
**Pull request by fwqaaq**

### CODE REVIEW

File: files/zh-cn/web/api/client/id/index.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+{{APIRef("Service Workers API")}}
+
+{{domxref("Client")}} 接口的 **`id`** 只读属性，返回 {{domxref("Client")}} 对象的通用唯一标识。
+
+## 值
+
+唯一标识对象的字符串。
+
+## 参考
```
Comment (yin1999) [inline files/zh-cn/web/api/client/id/index.md:14]:
```suggestion
## 规范
```

File: files/zh-cn/web/api/client/index.md [hunk 2]
```diff
@@ -3,21 +3,21 @@ title: Client
...
 ---
 
-{{SeeCompatTable}}{{APIRef("Service Workers API")}}
+{{APIRef("Service Workers API")}}
 
-`Client` 接口表示一个可执行的上下文，如{{domxref("Worker")}}或{{domxref("SharedWorker")}}。{{domxref("Window")}} 客户端由更具体的{{domxref("WindowClient")}}表示。你可以从{{domxref("Clients.matchAll","Clients.matchAll()")}} 和{{domxref("Clients.get","Clients.get()")}}等方法获取`Client`/`WindowClient`对象。
+`Client` 接口表示一个可执行的上下文，如 {{domxref("Worker")}} 或 {{domxref("SharedWorker")}}。{{domxref("Window")}} 客户端由更具体的 {{domxref("WindowClient")}} 表示。你可以从 {{domxref("Clients.matchAll","Clients.matchAll()")}} 和 {{domxref("Clients.get","Clients.get()")}} 等方法获取 `Client`/`WindowClient` 对象。
 
-## Methods
+## 方法
```
Comment (yin1999) [inline files/zh-cn/web/api/client/index.md:10]:
```suggestion
## 实例方法
```

File: files/zh-cn/web/api/client/index.md [hunk 3]
```diff
@@ -3,21 +3,21 @@ title: Client
...
 
 - {{domxref("Client.postMessage()")}}
   - : 向 client 发送一条消息。
 
-## Properties
+## 属性
```
Comment (yin1999) [inline files/zh-cn/web/api/client/index.md:15]:
```suggestion
## 实例属性
```

File: files/zh-cn/web/api/client/index.md [hunk 4]
```diff
@@ -3,21 +3,21 @@ title: Client
...
+{{APIRef("Service Workers API")}}
 
-`Client` 接口表示一个可执行的上下文，如{{domxref("Worker")}}或{{domxref("SharedWorker")}}。{{domxref("Window")}} 客户端由更具体的{{domxref("WindowClient")}}表示。你可以从{{domxref("Clients.matchAll","Clients.matchAll()")}} 和{{domxref("Clients.get","Clients.get()")}}等方法获取`Client`/`WindowClient`对象。
+`Client` 接口表示一个可执行的上下文，如 {{domxref("Worker")}} 或 {{domxref("SharedWorker")}}。{{domxref("Window")}} 客户端由更具体的 {{domxref("WindowClient")}} 表示。你可以从 {{domxref("Clients.matchAll","Clients.matchAll()")}} 和 {{domxref("Clients.get","Clients.get()")}} 等方法获取 `Client`/`WindowClient` 对象。
 
-## Methods
+## 方法
 
 - {{domxref("Client.postMessage()")}}
   - : 向 client 发送一条消息。
```
Comment (yin1999) [inline files/zh-cn/web/api/client/index.md:13]:
```suggestion
  - : 向客户端发送一条消息。
```

File: files/zh-cn/web/api/client/index.md [hunk 5]
```diff
@@ -3,21 +3,21 @@ title: Client
...
 
-## Properties
+## 属性
 
 - {{domxref("Client.id")}} {{readonlyInline}}
   - : 客户端的唯一通用标识符，字符串形式。
```
Comment (yin1999) [inline files/zh-cn/web/api/client/index.md:18]: 与 client/id 中的翻译不一致，统一一下

File: files/zh-cn/web/api/client/index.md [hunk 6]
```diff
@@ -3,21 +3,21 @@ title: Client
...
 - {{domxref("Client.type")}} {{readonlyInline}}
-  - : 客户端的类型，字符串形式。可能是 `"window"`, `"worker"`, 或 `"sharedworker"`。
+  - : 客户端的类型，字符串形式。可能是 `"window"`、`"worker"` 或 `"sharedworker"`。
```
Comment (yin1999) [inline files/zh-cn/web/api/client/index.md:20]:
```suggestion
  - : 表示客户端类型的字符串。可能是 `"window"`、`"worker"` 或 `"sharedworker"`。
```

File: files/zh-cn/web/api/client/index.md [hunk 7]
```diff
@@ -3,21 +3,21 @@ title: Client
...
 - {{domxref("Client.url")}} {{readonlyInline}}
   - : 客户端的 URL，字符串形式。
```
Comment (yin1999) [inline files/zh-cn/web/api/client/index.md:22]:
```suggestion
  - : 表示客户端 URL 的字符串。
```

**Review by yin1999 — COMMENTED** [review]
Some suggestions. You can apply all the changes with a single commit in code review (https://github.com/mdn/translated-content/pull/11351/files): click every `Add suggestion to batch` button and then `commit suggestions`

**Comment by fwqaaq** [comment]
> Some suggestions. You can apply all the changes with a single commit in code review (https://github.com/mdn/translated-content/pull/11351/files): click every `Add suggestion to batch` button and then `commit suggestions`

It's so cool

File: files/zh-cn/web/api/client/index.md [hunk 8]
```diff
@@ -3,23 +3,23 @@ title: Client
...
 
 - {{domxref("Client.postMessage()")}}
-  - : 向 client 发送一条消息。
+  - : 向客户端发送一条消息。
 
-## Properties
+## 实例属性
 
 - {{domxref("Client.id")}} {{readonlyInline}}
   - : 客户端的唯一通用标识符，字符串形式。
```
Comment (fwqaaq) [inline files/zh-cn/web/api/client/index.md:18]:
```suggestion
  - : 客户端作为字符串的唯一通用标识。
```

**Review by yin1999 — APPROVED** [review]

**Pull request merged**